### ISSUE TITLE
OSVDB-109206 Page not found

### ISSUE THREAD

**Issue by JuanitoFatas** [rubysec/ruby-advisory-db#222]
Hello!

I found that [OSVDB-109206.yml exists](https://github.com/rubysec/ruby-advisory-db/blob/master/gems/twitter-bootstrap-rails/OSVDB-109206.yml), but http://rubysec.com/advisories/OSVDB-109206 couldn't be found?

:confused: ?

**Comment by reedloden** [comment]
The CVE takes priority in that case, so http://rubysec.com/advisories/CVE-2014-4920/ exists.

Perhaps we should create pages for both? or use the name of the advisory file?

Personally, I think we should have our own numbering system and stop relying on CVE and OSVDB for assignments before we can post something.

**Comment by JuanitoFatas** [comment]
> Perhaps we should create pages for both?

Yes that's why I expected :smile: 

> use the name of the advisory file?

I thought we are using the name of the advisory file as part of URL? No?

CVE-2014-4920.yml =>  http://rubysec.com/advisories/CVE-2014-4920
OSVDB-109206.yml =>  http://rubysec.com/advisories/OSVDB-109206


> Personally, I think we should have our own numbering system and stop relying on CVE and OSVDB for assignments before we can post something.

That sounds good but do you think rely on CVE and OSVDB are easier to find and consume by scripts?

Thank you!

**Comment by reedloden** [comment]
No, we're actually ignoring the filename and using the CVE if it exists, otherwise the OSVDB ID.

```
      id   = if advisory['cve'] then "CVE-#{advisory['cve']}"
             else                    "OSVDB-#{advisory['osvdb']}"
             end
```

I'll see about creating pages for both for now (with a canonical URL set to only one).

As for the numbering, we would still ensure we list CVE and OSVDB identifiers, but we just won't be blocked on waiting for one of them in order to post an advisory.

**Comment by JuanitoFatas** [comment]
> I'll see about creating pages for both for now (with a canonical URL set to only one).

Thanks so much for this! No hurries : )


> As for the numbering, we would still ensure we list CVE and OSVDB identifiers, but we just won't be blocked on waiting for one of them in order to post an advisory.

This is a good idea, I am +1.

**Comment by mveytsman** [comment]
@reedloden I agree, we should have our own numbering system.

I'm starting a new issue about that

**Comment by JuanitoFatas** [comment]
Moved to #224, #251 also related.
